Modules/Quotas/src/Quotas.Application/Metrics/MetricStatusesService.cs

```diff
@@ -44,3 +44,8 @@ private static IEnumerable<MetricKey> ParseMetricKeys(IEnumerable<string> metric
         return parsedMetricKeys;
     }
 }
+
+public interface IMetricStatusesService
```

Comment:
Why the interface? Afaik FakeItEasy allows mocking classes as well (may be wrong though).

Reply:
I need either an interface or the method to be virtual or abstract.
Out of the options available, figured the interface was the best fit since we even discussed having it on the MetricStatusesService task.

Reply:
Okay, agree